decider-bcl2fastq/src/main/java/ca/on/oicr/pde/deciders/Bcl2fastqDecider.java

```diff
@@ -11,8 +11,8 @@
 import ca.on.oicr.pde.deciders.configuration.StudyToOutputPathConfig;
 import ca.on.oicr.pde.deciders.data.Barcode;
 import ca.on.oicr.pde.deciders.data.BarcodeCollision;
-import ca.on.oicr.pde.deciders.handlers.Bcl2Fastq1Handler;
-import ca.on.oicr.pde.deciders.handlers.Bcl2Fastq2Handler;
+import ca.on.oicr.pde.deciders.handlers.Bcl2Fastq_2_7_1_Handler;
+import ca.on.oicr.pde.deciders.handlers.Bcl2Fastq_2_9_1_Handler;
```

Comment:
Rather than referring to them by their workflow version, it would make more sense to use the bcl2fastq version, like you had before. Why did you decide to use the workflow version?

Reply:
The decider supports scheduling workflow runs for specific workflow versions.  Workflow versions 2.7.1 and 2.9.1 do not support "provision_out_undetermined=false".  If "--do-not-provision-undetermined" and workflow type CASAVA-2.7.1/2.9.1 are specified, an error is raised.